src/ISA/ISA/ArcTypes/CompositeCell.fs

```diff
@@ -35,6 +35,28 @@ type CompositeCell =
     member this.isTerm = match this with | Term _ -> true | _ -> false
     member this.isFreeText = match this with | FreeText _ -> true | _ -> false
 
+    /// <summary>
+    /// Updates current CompositeCell with information from OntologyAnnotation.
+    ///
+    /// For `Term`, OntologyAnnotation (oa) is fully set. For `Unitized`, oa is set as unit while value is untouched.
+    /// For `FreeText` oa.NameText is set.
+    /// </summary>
+    /// <param name="oa"></param>
+    member this.UpdateWithOA(oa:OntologyAnnotation) =
```

Comment:
This function seems a bit unintuitive. In what case would you either set the value OR the unit with a given input?